app/src/main/res/layout/activity_main.xml

```diff
@@ -87,13 +87,13 @@
         app:layout_constraintVertical_bias="0.073" />
 
     <TextView
-        android:id="@+id/textView5"
-        android:layout_width="380dp"
-        android:layout_height="50dp"
-        android:textSize="12sp"
-        android:layout_marginLeft="10dp"
-        android:layout_marginRight="10dp"
-        android:text="For verification, an SMS will be sent from your Mobile Number. Standard operator charges may apply. By proceeding you agree to our Terms and Conditions and Privacy Policy"
+        android:id="@+id/tvInfo"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/ht1"
```

Comment:
@raghav32 Please use wrap_content if view is to take as much space as of the text